.github/workflows/ubuntu.yml

```diff
@@ -49,6 +49,53 @@ jobs:
         cmake --build build_RZ -j 2
         ./build_RZ/bin/warpx Examples/Physics_applications/laser_acceleration/inputs_rz
 
+  build_2D_rz:
```

Comment:
This all looks good. Though, this name should presumably be "build_1D_2D". For consistency, perhaps the name above should be "build_cxxminimal_3D_RZ"?

Reply:
Good point :)

Reply:
(not minimal because minimal is with respect to third-party dependencies)